spec/index.html

```diff
@@ -5493,9 +5579,9 @@ <h5>IF</h5>
               <a href="#ebv">effective boolean value</a>, then returns the value of
               <code>expression2</code> if the EBV is true, otherwise it returns the value of
               <code>expression3</code>. Only one of <code>expression2</code> and
-              <code>expression3</code> is evaluated. If evaluating the first argument raises an error,
-              then an error is raised for the evaluation of the <code>IF</code> expression.</p>
-            <p>Examples: Suppose ?x = 2, ?z = 0 and ?y is not bound in some query solution:</p>
+              <code>expression3</code> is evaluated. If evaluating the first argument raises an [=error=],
+              then an [=error=] is raised for the evaluation of the <code>IF</code> expression.</p>
+            <p>Examples: Suppose that `?x = 2` and `?z = 0`, and that `?y` is not bound, in some query solution:</p>
```

Comment:
It *might* be worth changing the `=` signs here to `is bound to`, i.e.,
```suggestion
            <p>Examples: Suppose that `?x` is bound to `2`, `?z` is bound to `0`,
              and `?y` is not bound, in some query solution:</p>
```

Reply:
I prefer the existing form.

Reply:
I prefer Ted's edit suggestion.

Reply:
Change made.

(The original change was merely following through on the group's wish for reformatting.)